tests/cases/fourslash/extract-method1.ts

```diff
@@ -14,18 +14,18 @@
 
 goTo.select('start', 'end')
 edit.applyRefactor({
-    refactorName: "Extract Method",
-    actionName: "scope_0",
+    refactorName: "Extract Symbol",
+    actionName: "function_scope_1",

```

Comment:
>1 [](start = 32, length = 1)

0 is now the method itself.  We won't insert a function there, but we will insert a local.